Setting tcp_listen_port in /etc/audit/auditd.conf to listen for audit
records from remote systems results in auditd failing to start:

# auditd -f
....
Init complete, auditd 1.7.13 listening for events (startup state enable)
Cannot bind tcp listener socket to port 60
The audit daemon is exiting.

Looking at the strace:

bind(6, {sa_family=0x200 /* AF_??? */,
sa_data="\0<\0\0\0\0\0\0\0\0\0\0\0\0"}, 16) = -1 EAFNOSUPPORT (Address family not supported by protocol)

--- a/audisp/plugins/remote/audisp-remote.c
+++ b/audisp/plugins/remote/audisp-remote.c
@@ -833,7 +833,7 @@ static int init_sock(void)
 		struct sockaddr_in address;
 		
 		memset (&address, 0, sizeof(address));
-		address.sin_family = htons(AF_INET);
+		address.sin_family = AF_INET;
 		address.sin_port = htons(config.local_port);
 		address.sin_addr.s_addr = htonl(INADDR_ANY);
 
--- a/src/auditd-listen.c
+++ b/src/auditd-listen.c
@@ -870,7 +870,7 @@ int auditd_tcp_listen_init ( struct ev_l
 			(char *)&one, sizeof (int));
 
 	memset (&address, 0, sizeof(address));
-	address.sin_family = htons(AF_INET);
+	address.sin_family = AF_INET;
 	address.sin_port = htons(config->tcp_listen_port);
 	address.sin_addr.s_addr = htonl(INADDR_ANY);
 
